### DISCUSSION TITLE
Desenvolver calendário de eventos especiais fixo

### DISCUSSION THREAD

**Discussion by austv** [leogianfagna/IssueTracker#933] [Ideia]
### Descrição da tarefa

Eventos mais importantes que acontecem de forma mais especial (não sazonal):
- Caça ao dragão
- Super Pinhata
- Corrida Maluca (novidade a cada realização)

Desenvolver em que dia eles vão acontecer, assim como assinar a tarefa para alguém sempre trazer coisa nova para o corrida maluca, além do que, desenvolver a ideia de cada um deles.

**Comment by leogianfagna** [comment]
Caça ao dragão usar mcfuncions, para ter uma dragão completamente diferente e drops de muitos itens.
